cmd/docker-app/image-add.go

```diff
@@ -67,8 +81,11 @@ subdirectory.`,
 	}
 	if internal.Experimental == "on" {
 		cmd.Flags().StringArrayVarP(&imageAddComposeFiles, "compose-files", "c", []string{}, "Override Compose files")
-		cmd.Flags().StringArrayVarP(&imageAddSettingsFile, "settings-files", "s", []string{}, "Override settings files")
-		cmd.Flags().StringArrayVarP(&imageAddEnv, "env", "e", []string{}, "Override environment values")
 	}
+	cmd.Flags().StringArrayVarP(&imageAddSettingsFile, "settings-files", "f", []string{}, "Override settings files")
+	cmd.Flags().StringArrayVarP(&imageAddEnv, "set", "s", []string{}, "Override environment values")
```

Comment:
I think we should factorize in a follow up all these flags for all commands rendering the compose file.